### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by Drewbi**
Adds complete support for dark mode toggling

### CODE REVIEW

**Comment by JeremiahPinto** [comment]
This isn't complete. None of the user pages are done, i.e. membership, dashboard, account_settings. Plus the theme icon needs a tooltip for users and `aria` text for a11y

**Comment by Drewbi** [comment]
@JeremiahPinto No it complete

**Comment by JeremiahPinto** [comment]
> Plus the theme icon needs a tooltip for users and `aria` text for a11y.
 u read this @Drewbi? Plus the map no longer has the 3d layer

File: lib/theme.ts [hunk 1]
```diff
@@ -38,6 +40,7 @@ export interface Theme {
   colors?: {
     primary?: string
     secondary?: string
+    darkBg?: string
```
Comment (JeremiahPinto) [inline lib/theme.ts:43]: wheres lightBg?

File: components/Utils/Footer/index.tsx [hunk 2]
```diff
@@ -43,6 +43,7 @@ const Footer = () => {
             </div>
             <p className='monospace m-0'>
               <small>Made with &#10084;</small>
+              <small css={{ color: 'black' }}> by Jeremiah</small>
```
Comment (JeremiahPinto) [inline components/Utils/Footer/index.tsx:46]: Also whut

File: components/Utils/Footer/SecurityModal/index.tsx [hunk 3]
```diff
@@ -1,15 +1,20 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
```
Comment (JeremiahPinto) [inline components/Utils/Footer/SecurityModal/index.tsx:2]: y u importing the jsx if u not using emotion for this modal

Comment (JeremiahPinto) [reply components/Utils/Footer/SecurityModal/index.tsx:2]: as with all the footer modals

**Review by JeremiahPinto — CHANGES_REQUESTED** [review]
fixes needed. Plus it no work properly on chrome

File: components/Utils/Footer/index.tsx (hunk 2 above)
Comment (Drewbi) [reply components/Utils/Footer/index.tsx:46]: It easter egg

File: components/Utils/Footer/SecurityModal/index.tsx (hunk 3 above)
Comment (Drewbi) [reply components/Utils/Footer/SecurityModal/index.tsx:2]: Bc it fixed the react thing that was complaining and stopping my commits from going through

File: components/Utils/Footer/index.tsx (hunk 2 above)
Comment (JeremiahPinto) [reply components/Utils/Footer/index.tsx:46]: use `text-primary` if u want to use this

Comment (Drewbi) [reply components/Utils/Footer/index.tsx:46]: Didn't work when I tried it

**Review by JeremiahPinto — APPROVED** [review]

**Pull request merged**